app/globals.css

```
@@ -291,10 +317,31 @@ body {
-  .dark .react-flow__node.selected .border-teal-500 {
+  .dark .react-flow__node.selected .border-teal-500,
+  .dark .react-flow__node.selected .border-teal-600 {
     box-shadow: 0 0 0 2px rgba(45, 212, 191, 0.65), 0 0 18px rgba(45, 212, 191, 0.7), 0 0 40px rgba(45, 212, 191, 0.4), inset 0 0 0 1px rgba(255, 255, 255, 0.18) !important;
   }
 
+  .dark .react-flow__node.selected .dark\\:border-indigo-400,
+  .dark .react-flow__node.selected .border-indigo-500 {
+    box-shadow: 0 0 0 2px rgba(129, 140, 248, 0.65), 0 0 18px rgba(129, 140, 248, 0.7), 0 0 40px rgba(129, 140, 248, 0.4), inset 0 0 0 1px rgba(255, 255, 255, 0.18) !important;
+  }
+
+  .dark .react-flow__node.selected .dark\\:border-violet-400,
```

Biome (2.1.2)

[error] Unexpected unknown pseudo-class border-violet-400 

See MDN web docs for more details.

(lint/correctness/noUnknownPseudoClass)